test/test_twodturb.jl

```diff
@@ -172,7 +170,7 @@ function test_twodturb_advection(dt, stepper; n=128, L=2π, nu=1e-2, nnu=1, mu=0
     nothing
   end
 
-  prob = TwoDTurb.Problem(nx=n, Lx=L, nu=nu, nnu=nnu, mu=mu, nmu=nmu, dt=dt, stepper=stepper, calcF=calcF!, stochastic=false)
+  prob = TwoDTurb.Problem(nx=n, Lx=L, nu=nu, nnu=nnu, mu=mu, nmu=nmu, dt=dt, stepper=stepper, calcF=calcF!, stochastic=false, dev=dev)
```

Comment:
Should we just use greek for things like nu and mu? It's 2019 after all.

Reply:
hm... we've been going back and forth on this.
you know I'm prejudiced in favor of greek symbols so I'd say "yes".

Reply:
We want to be on the right side of history.

Reply:
see eaa0f12